### PR TITLE
Api  detail data for movie and person

### PR DESCRIPTION
**Pull request by damdlani**

### CODE REVIEW

File: src/features/movies/MovieCredits/getMovieCredits.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import {API_KEY, URL_ADDRESS} from "../../../lib/consts";
+
+export const getMovieCredits = async (movie_id = 337401) => {
```
Comment (anetaszynal) [inline src/features/movies/MovieCredits/getMovieCredits.js:3]: We have to remember about passing this argument from up

**Review by anetaszynal — APPROVED** [review]
Great job! :)

File: src/features/movies/MovieDetail/movieDetailSaga.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import {call, put, takeLatest, delay} from "redux-saga/effects";
+import {getMovieDetail} from "./getMovieDetail";
+import {fetchMovieDetail, 
+        fetchMovieDetailSuccess,
+        fetchMovieDetailError
+        } from "./movieDetailSlice";
+
+function* fetchMovieDetailHandler() {
+    try {
+        yield delay(1000)
```
Comment (MikeAdamczyk) [inline src/features/movies/MovieDetail/movieDetailSaga.js:10]: Do we need delay(1000) :) ?

**Review by MikeAdamczyk — APPROVED** [review]
looks good 😃

**Pull request merged**